src/modal-load-recipe.ts

```diff
@@ -16,6 +23,13 @@ export class LoadRecipeModal extends Modal {
 
 		new Setting(contentEl).setName("url: ").addText((text) => {
 			text.setPlaceholder("https://www.example.com/recipe");
+
```

Comment:
Added this back in to just auto-add a recipe url for development and testing. If this is a bad idea, happy to pull it back out..

Reply:
It's probably better not to have this, otherwise it makes it impossible to turn on debug mode to see errors being thrown with a particular URL (e.g. like what happened for #31). Or at least have it a editable text field setting to be able to test with different URLs as needed.

Reply:
Thinking the same thing. I’ll either remove, comment it out or make it a setting like you suggested. Thanks!